Makefile

```diff
@@ -197,15 +215,14 @@ else
 	docker push gcr.io/gadget-core-production/dateilager:latest
 endif
 
-upload-prerelease-container-image: release
-	docker build -t gcr.io/gadget-core-production/dateilager:$(GIT_COMMIT) .
-	docker push gcr.io/gadget-core-production/dateilager:$(GIT_COMMIT)
+upload-prerelease-container-image:
+	docker build --platform linux/arm64,linux/amd64 --push -t us-central1-docker.pkg.dev/gadget-core-production/core-production/dateilager:pre-$(GIT_COMMIT) .
 
 build-local-container:
-	docker build -t dl-local:latest .
+	docker build --load -t dl-local:dev .
```

Comment:
kubernetes will always pull an image with the `latest` tag, which we don't want to do, cause it can't be pulled from anywhere. we want it to just sit in the local image cache, so we name it something else